model/textparse/openmetricsparse.go

```diff
@@ -467,6 +580,25 @@ func (p *OpenMetricsParser) parseLVals(offsets []int, isExemplar bool) ([]int, e
 	}
 }
 
+// skipCTParsing checks if the current series is a _created series and skips it if necessary.
+func (p *OpenMetricsParser) skipCTParsing(e Entry, err error) (Entry, error) {
+	if e != EntrySeries || !p.skipCTSeries || err != nil {
```

Comment:
This is not ideal, bit too much validation for little benefit:
* we should rule out errors before this method - we do this for all other functions/methods
* parseSuffix is only called for entryseries, so unless there was an error it's a series, we could build a flow that assumes this safely.

Perhaps better way would be to have function `isCreatedSeries()` and then do in main loop

```
if p.isCreatedSeries() && p.skipCTSeries {
    return p.Next()
}
```

.. so again, we do recursion in the most explicit place

Reply:
done